stonadsstatistikk-ef/src/main/kotlin/no/nav/familie/eksterne/kontrakter/ef/BehandlingDVH.kt

```diff
@@ -258,3 +259,11 @@ data class ÅrsakRevurdering(
     val opplysningskilde: String,
     val årsak: String
 )
+
+enum class AvslagÅrsak {
```

Comment:
Til info, jeg har sendt opplysningskilde og årsak som strings, litt for å unngå å trenge å oppdatere flere stedet når vi endrer kontraktet hos oss :)

Reply:
Hvis enum skal brukes så kan det være fint å legge in det i `FellesTilEksterneKontrakterEnumTest` i iverksett :)

Reply:
Godt poeng - kjører på med strings da